docs/strategy-customization.md

```diff
@@ -532,6 +532,27 @@ If you want to use a strategy from a different directory you can pass `--strateg
 freqtrade trade --strategy AwesomeStrategy --strategy-path /some/directory
 ```
 
+### Derived strategies
+
+The strategies can be derived from other strategies. This avoids duplication of your custom strategy code. You can use this technique to override small parts of your main strategy, leaving the rest untouched:
+
+``` python
+class MyAwesomeStrategy(IStrategy):
+    ...
+    stoploss = 0.13
+    trailing_stop = False
+    # All other attributes and methods are here as they
+    # should be in any custom strategy...
+    ...
+
+class MyAwesomeStrategy2(MyAwesomeStrategy):
+    # Override something
+    stoploss = 0.08
+    trailing_stop = True
+```
+
+Both attributes and methods may be overriden, altering behavior of the original strategy in a way you need. The strategy classes may be located in the same module (python file with the source code of your strategy) or in different modules (different python files). In the latter case you need to properly import the strategy class you derive the new one from.
```

Comment:
> or in different modules (different python files). In the latter case you need to properly import the strategy class you derive the new one from.

I understand it works (technically), and i can get it to work if i must - but it needs some more than just 2 files to get this to work. 
It needs to be in a package (folder containing `__init__.py`) - otherwise the import won't work correctly - which i think makes this a hurdle for non-python developers.
I'm also not 100% certain if relative imports (`from .awesomestrategy import MyAwesomeStrategy`) would work with our import method (and i did not try yet).

Now i'm sceptical that the everyday trader user will get this to work ...

Reply:
so should I remove this part?
I'm not sure python3 needs empty `__init__.py`...

Reply:
to import relatively? 
I'm pretty sure it does as i've been fighting that at work today...

Reply:
okay, I removed this to keep the docs simple